source/agora/test/Base.d

```diff
@@ -58,6 +58,9 @@ public class TestNetworkManager : NetworkManager
     /// Also kept here to avoid any eager garbage collection.
     public RemoteAPI!TestAPI[PublicKey] apis;
 
+    /// Workaround compiler bug that triggers in `std.concurrency`
+    protected __gshared std.concurrency.Tid[string] tbn;
```

Comment:
Is there an associated tracking bug for this? It would be useful to link it here.

Reply:
No because I don't have simple steps to reproduce.